iset.mm

```diff
@@ -111891,6 +111891,26 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) , CC )
       CXJMPUAOZMPSXJYCLUGMPAXJYCWDVOWEVH $.
   $}
 
+  ${
+    $d k v F $.  $d k v M $.  $d k v N $.  $d k v ph $.
+    fsum3ser.1 $e |- ( ( ph /\ k e. ( ZZ>= ` M ) ) -> ( F ` k ) = A ) $.
+    fsum3ser.2 $e |- ( ph -> N e. ( ZZ>= ` M ) ) $.
+    fsum3ser.3 $e |- ( ( ph /\ k e. ( ZZ>= ` M ) ) -> A e. CC ) $.
+    $( A finite sum expressed in terms of a partial sum of an infinite series.
+       The recursive definition of follows as ~ fsum1 and ~ fsump1 , which
```

Comment:
of ?

Reply:
"of ...?" also in the comment of the next theorem. It was/is already unclear in the comment of the original theorem ~fsumser in set.mm.

Reply:
Here I'll just delete the "of" in all three places.